### PR TITLE
Added support of docker-compose for local setup.

### PR DESCRIPTION
**Pull request by shubhamdang**

### CODE REVIEW

**Review by ColonelPanicks — CHANGES_REQUESTED** [review]
This provides an alternative method for those that do not wish to install the Python dependencies directly on their system (although Docker dependencies would be required on the system). 

Before merging this I think the `README.md` should be updated to describe this as an alternative `Setup` method to the standard of just pip installing the packages needed. Please can you update the README in this branch to reflect these updates.

**Comment by shubhamdang** [comment]
> This provides an alternative method for those that do not wish to install the Python dependencies directly on their system (although Docker dependencies would be required on the system).
> 
> Before merging this I think the `README.md` should be updated to describe this as an alternative `Setup` method to the standard of just pip installing the packages needed. Please can you update the README in this branch to reflect these updates.

Changes have been done

File: README.md [hunk 1]
```diff
@@ -25,6 +27,57 @@ A test of `mkdocs` for hosting the OpenFlight docs
...
 
+### Container
+- Prerequisite
+  `docker and docker-compose`
+
+- Start the application
+  ```bash
+  # To run in foregorund
+  cd docs.openflighthpc.org
+  docker-compose up -d
```
Comment (ColonelPanicks) [inline README.md:38]: This looks the same as the "run in background" command, is this as expected?

File: README.md [hunk 2]
```diff
@@ -25,6 +27,57 @@ A test of `mkdocs` for hosting the OpenFlight docs
...
+  cd docs.openflighthpc.org
+  docker-compose down
+  ```
+
+- Access application
+  ```bash
+  http://<machine_ip>:8000
+  ```
+  
+- Run application other 8000
```
Comment (ColonelPanicks) [inline README.md:60]:
```suggestion
- Run application on a different port
```

File: README.md (hunk 1 above)
Comment (shubhamdang) [reply README.md]: thanks, Updated

File: README.md (hunk 2 above)
Comment (shubhamdang) [reply README.md]: thanks, Updated

**Pull request merged**